upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -484,6 +487,10 @@ func (c *ApplyClusterCmd) Run() error {
 			stageAssetsLifecycle = lifecyclePointer(fi.LifecycleExistsAndWarnIfChanges)
 			iamLifecycle = lifecyclePointer(fi.LifecycleExistsAndWarnIfChanges)
 			networkLifecycle = lifecyclePointer(fi.LifecycleExistsAndWarnIfChanges)
+		} else if c.PhaseValidationPolicy == fi.LifecycleValidationLoose {
+			stageAssetsLifecycle = lifecyclePointer(fi.LifecycleIgnore)
```

Comment:
Not really clear why we're ignoring assets, but it is consistent with the existing `else` branch